src/components/App.css

```diff
@@ -0,0 +1,52 @@
+*{
```

Comment:
under the src folder create an asset folder 
where you can have your /css folder where you will be putting all styles and any other folders

such as /images ....